package.json

```diff
@@ -77,7 +77,9 @@
   "scripts": {
     "build:prod": "webpack --config webpack.prod.js --mode production",
     "build": "webpack --config webpack.dev.js --mode development",
-    "watch": "webpack --config webpack.dev.js --watch --mode development"
+    "watch": "webpack --config webpack.dev.js --watch --mode development",
+    "lint": "eslint apps adhocracy-plus/assets --ext .js,.jsx && stylelint 'adhocracy-plus/assets/scss/**/*.scss' --syntax scss",
+    "lint-staged" : "eslint apps adhocracy-plus/assets --ext .js,.jsx --fix-dry-run"
```

Comment:
This looks to me like it would only defer from `lint` in that in adds `--fix-dry-run` and doesn't run `stylelint`, but I don't see it only linting staged files :/